porthtml.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+
+  <title>ibrahim Portfolio</title>
+ 
+ 
+</head>
+
+<body class="body1">
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li class="nav"><a href="#">my logo </a></li>
+        <li class="pac"><a href="#contact">Contact</a></li>
+        <li class="pac"><a href="#PORTFOLIO">Portfolio</a></li>
+        <li class="pac"><a href="#about">about</a></li>
+        <li id="ham">
+          <img src="Union.svg" alt="menu image" class="menu-image ham" />
+        </li>
+      </ul>
+    </nav>
+    <div id="nav-two" class="nav-two">
+      <ul id="main-ls">
+        <li><a href="#project-list"></a>Contact</li>
+        <li><a href="#about">Portfolio</a></li>
+        <li><a href="#contact">ABOUT</a></li>
+      </ul>
+    </div>
+  </header>
+  <main class="container1">
+    <div class="con"></div>
+    <section class="sec1">
```

Comment:
- [ ] Please make sure you follow the design as much as possible and include the background images as in the given template for this section.